astropy/units/astrophys.py

```diff
@@ -32,7 +32,7 @@
          doc="parsec: approximately 3.26 light-years.")
 
 def_unit(['solRad', 'R_sun', 'Rsun'], _si.R_sun, namespace=_ns,
-         doc="Solar radius", prefixes=True,
+         doc="Nominal Solar radius(hence no uncertainty)", prefixes=True,
```

Comment:
I think the "(hence no uncertainty)" isn't needed here (or even if it is, there should be a space before the open parens).  But if anything this nomenclature can be clarified in the docs somewhere in docs/constants.